app/components/table/TableBodyMetrics.tsx

```diff
@@ -35,23 +86,35 @@ export default function TableBodyPrivateRequests({ rows, resource, title }: Tabl
         <TableHeader title={title} />
         <div className="divide-y divide-grey-200/5">
           {rows.map((row, index) => (
-            <div key={row.name}>
+            <div key={row.podName}>
               <div
                 className={`hover:bg-gray-100 transition-colors duration-200 grid grid-cols-1 md:grid-cols-6 lg:grid-cols-12 gap-4 px-4 py-3 sm:px-6 lg:px-8 ${
                   index === 0 && 'bg-gray-100'
                 }`}
               >
-                <div className="md:col-span-2 lg:col-span-6">
-                  <TruncatedTooltip label={row.name}>
-                    <span className={`${index === 0 && 'font-bold'}`}>{_truncate(row.name, { length: 100 })}</span>
+                <div className="md:col-span-1 lg:col-span-3">
+                  <TruncatedTooltip label={row.podName}>
+                    <span className={`${index === 0 && 'font-bold'}`}>{_truncate(row.podName, { length: 100 })}</span>
                   </TruncatedTooltip>
                 </div>
-                <div className={`md:col-span-1 lg:col-span-2 ${index === 0 && 'font-bold'}`}>{row.usage[resource]}</div>
+                <div className="md:col-span-1 lg:col-span-3">
+                  <TruncatedTooltip label={row.containerName}>
+                    <span className={`${index === 0 && 'font-bold'}`}>
+                      {_truncate(row.containerName, { length: 100 })}
+                    </span>
+                  </TruncatedTooltip>
+                </div>
+                <div className={`md:col-span-1 lg:col-span-2 ${index === 0 && 'font-bold'}`}>
```

Comment:
please use `classnames` for readability. and create a Component above to reduce the repeats.